server/index.js

```diff
@@ -26,6 +26,6 @@ app.use('/api/user', routesUser)
 app.use('/api/category', routesCategory)
 app.use('/api/visualization', routesVisualization)
 
-app.listen(3000, () => {
-    console.log(`Server Started at ${3000}`)
+app.listen(3001, () => {
+    console.log(`Server Started at ${3001}`)
 })
```

Comment:
Did @NirmalAgnihotri explicitly choose 4-space indents, or do you all not have Prettier installed? You also have to set VS Code to use Prettier as the auto formatter

Reply:
The PR to add linting and formatting just went in, I've let the devs know that main has to be merged into this branch before it can be merged in, which will reject this branch. But once they run they run the prettier, indents will be set to 2 spaces. So they can run it before merging, or they can install and keep it as auto-formatting. The latter is easier on them, but in the end the result in the repo is the same.